Securitytxt/Model/Config/Backend/Validate.php

```diff
@@ -123,4 +147,85 @@ private function validateUrlField(string $fieldName, string $fieldValue): void
             );
         }
     }
+
+    /**
+     * Get Value from form or inheriting value.
+     *
+     * @param array $fieldData
+     * @return string
+     */
+    private function getDataValue(array $fieldData): string
+    {
+        $result = '';
+        if (isset($fieldData['value'])) {
+            $result = $fieldData['value'];
+        }
+
+        return $result;
+    }
+
+    /**
+     * Check exists value data
+     *
+     * @param array $fieldData
+     * @return bool
+     */
+    private function existDataValue(array $fieldData): bool
+    {
+        if (isset($fieldData['value'])) {
+            if ($fieldData['value'] !== '' || empty($fieldData['value'])) {
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+    /**
+     * Check is Empty value
+     *
+     * @param string $key
+     * @param array $fieldData
+     * @return bool
+     */
+    private function isEmptyValue(string $key, array $fieldData): bool
+    {
+        if ($this->existDataValue($fieldData[$key]) && $this->getDataValue($fieldData[$key]) === '') {
+            return true;
+        }
+
+        return false;
```

Comment:
return ($this->existDataValue($fieldData[$key]) && $this->getDataValue($fieldData[$key]) === '');